VATSpy.dat

```diff
@@ -18069,16 +18069,18 @@ NFFF|Nadi Oceanic||NFFF
 NFFN|Nadi||NFFN
 NVVV|Port Vila||NVVV
 NSTU|Pago Pago||NZZO
-NTTT|Tahiti Oceanic||NTTT
+NTTT|Tahiti Radio||NTTT

```

Comment:
We aim to only have the FIR names here when possible and as this is the only FIR that starts with "NT" I would suggest edit line 120 to include the "Radio" instead, like this:

> French Polynesia|NT|Radio

And then only have "Tahiti" as the FIR name.

Reply:
That makes sense, however we intend to add NTTT specific sector splits in the future, so this wouldn't work. This is the preferred solution.